Detector/DetFCChhBaseline1/compact/FCChh_DectMaster.xml

```diff
@@ -32,9 +32,8 @@
   <include ref="./FCChh_Solenoids.xml" />
   <include ref="./FCChh_Shielding.xml" />
 
-
   <include ref="../../DetFCChhTrackerTkLayout/compact/Tracker.xml" />
-  <include ref="../../DetFCChhECalSimple/compact/FCChh_ECalBarrel_Mockup.xml" />
+  <include ref="../../DetFCChhECalInclined/compact/FCChh_ECalBarrel_withCryostat.xml" />
```

Comment:
For overlaps no (but it's not going to be changed anymore).
There are few other tests using its implementation. It's possible it will be removed soon from those as well (e.g. once we decide on noise in the inclined geometry).